thamos/cli.py

```diff
@@ -381,6 +382,7 @@ def advise(
             debug=debug,
             nowait=no_wait,
             force=force,
+            source_type=ThothAdviserIntegrationEnum.CLI.value
```

Comment:
BTW this should be:
```python
            source_type=ThothAdviserIntegrationEnum.CLI
```

to conform signature:

```python
    source_type: typing.Optional[ThothAdviserIntegrationEnum] = None,
```

And in the advise method:

```python
        "source_type": source_type.name,

```

Reply:
Updated and also added check on s2i using the method.